### DISCUSSION TITLE
How to automatically lower screen brightness to 5% using "light -S 5" after inactivity and change the brightness back to the previous value after any input?

### DISCUSSION THREAD

**Discussion by ImGGAAVVIINN** [i3/i3#5446] [Q&A]
I am on a laptop and I would like to make i3wm set the brightness to 5% using the command:

light -S 5

And I want to set it back to the previous brightness I am on after I move my mouse or press my keyboard.

**Comment by okraits** [comment]
You can execute any command at i3 startup by using the `exec` command in the config.

However, i3 doesn't monitor inactivity, it's a window manager.
